README.md

```diff
@@ -16,34 +16,45 @@ Documentation can be found [here](https://eosio.github.io/eosjs)
 
 ## Basic Usage
 
+### Browser 
+
+Importing using ES6 module syntax in the browser is supported if you have a transpiler, such as Babel.
+```js
+import { Api, JsonRpc, RpcError, JsSignatureProvider } from 'eosjs';
+```
+
 ### NodeJS
 
+Importing using commonJS syntax is supported by node out of the box.
 ```js
-const eosjs = require('eosjs');
+const { Api, JsonRpc, RpcError, JsSignatureProvider } = require('eosjs');
 const fetch = require('node-fetch');                            // node only; not needed in browsers
 const { TextDecoder, TextEncoder } = require('text-encoding');  // node, IE11 and IE Edge Browsers
 ```
 
 ### SignatureProvider
```

Comment:
Should this header be `JsSignatureProvider` now and also referenced as `JsSignatureProvider` right below too?